### ISSUE TITLE
Pointcharge gradient

### ISSUE THREAD

**Issue by RagnarB83** [grimme-lab/xtb-python#55]
Is it possible to get access to the pointcharge gradient in the API ? I could not find it.

**Comment by awvwgk** [comment]
Duplicate of https://github.com/grimme-lab/xtb/issues/488

**Comment by RagnarB83** [comment]
I see that the pointcharge gradient is now available in the C-API, so it would be straightforward to use it it in the Python-API.
But this requires bumping the version from 6.4.1. to 6.5.1. 
Is that planned ?

**Comment by awvwgk** [comment]
Indeed, the CFFI should already generate a low-level interface if the prototype is available in the header. Happy to accept a patch for bumping the `xtb` version and implementing a new Python wrapper.
